src/components/Challenges/MultipleChoiceChallenge/MultipleChoiceChallenge.tsx

```diff
@@ -72,20 +80,31 @@ export const MultipleChoiceChallenge = ({
         Select the correct option:
       </Text>
       <StepMarkdown>{prompt}</StepMarkdown>
-      {options
-        ?.filter(removeEmpty)
-        .map(({ incorrectChoiceExplanation, text }, index) => (
-          <MultipleChoiceChallengeOption
-            text={text}
-            incorrectChoiceExplanation={incorrectChoiceExplanation}
-            isOptionSelected={isOptionSelected(index)}
-            onClickOption={() => onClickOption(index)}
-            showOptionIncorrectExplanation={shouldShowOptionIncorrectExplanation(
-              index,
-            )}
-            key={text}
-          />
-        ))}
+      {!showOptions && (
+        <Button
+          colorScheme="blue"
+          size="md"
+          marginBottom="40px"
+          onClick={() => setShowOptions(!showOptions)}
+        >
+          Show options
+        </Button>
+      )}
+      {showOptions &&
+        options
+          ?.filter(removeEmpty)
+          .map(({ incorrectChoiceExplanation, text }, index) => (
+            <MultipleChoiceChallengeOption
+              text={text}
+              incorrectChoiceExplanation={incorrectChoiceExplanation}
+              isOptionSelected={isOptionSelected(index)}
+              onClickOption={() => onClickOption(index)}
+              showOptionIncorrectExplanation={shouldShowOptionIncorrectExplanation(
```

Comment:
a ternary would be cleaner here than two && statements

Reply:
Fixed!